### PR TITLE
A fix to all the nits brought up from @toolness as he reviewed the merges

### PR DESCRIPTION
**Pull request by rossbruniges**
@toolness - can you check over these to confirm?

Thanks very much!

### CODE REVIEW

**Comment by toolness** [comment]
This looks great, thanks ross!

**Comment by rossbruniges** [comment]
@andrewhayward how's it looking to you?

File: gameon/users/middleware.py [hunk 1]
```diff
@@ -1,16 +1,29 @@
...
         return ('users_edit', 'django.views.static.serve', 'users_signout')
 
     def is_safe(self, path):
+        """
+        Checks the current request path is in the safe list above and if so
+        ignores it and returns as normal
+        """
         try:
             match = resolve(path)
             return match.url_name in self.__class__.safe_paths()
```
Comment (andrewhayward) [inline gameon/users/middleware.py:29]: Appreciate this is not a new addition, but the `__class__` here is redundant. Calling class methods from within class instances should resolve the method correctly, without needing to do it yourself. Unsure how I feel about static data behind function calls, but that's a discussion for another time.

File: gameon/users/middleware.py [hunk 2]
```diff
@@ -18,17 +31,20 @@ def is_safe(self, path):
...
+        """
+        if it's a request for the django debug toolbar AND we're in dev we can
+        ignore - this check now only applies to when the site is in dev
+        """
+        if settings.DEBUG and '__debug__' in request.path:
             return
         if self.is_safe(request.path):
             return
+        # remove the locale string - resolve won't work with it included
         path = u'/%s' % ('/'.join(request.path.split('/')[2:]),)
```
Comment (andrewhayward) [inline gameon/users/middleware.py:43]: Is it safe to assume that there will always be a locale string on the front of paths?

Comment (rossbruniges) [inline gameon/users/middleware.py:43]: I think so - it's pretty baked into Playdoh

File: gameon/users/middleware.py (hunk 1 above)
Comment (rossbruniges) [inline gameon/users/middleware.py:29]: This code has been through the ringer on so many projects now I think I'm going to leave as is . If we can make it better for the future though that would be awesome!

**Pull request merged**